robocop/config/config.py

```diff
@@ -31,6 +39,7 @@ def __init__(self):
         self.paths = []
         self.include_patterns = list()
         self.exclude_patterns = list()
+        self.filetypes = {'.robot', '.resource'}
```

Comment:
.robot and .resource formats are supported by default but RF still supports .tsv and .rst or .rest if they are indicated explicitly. It would be good to support them in the future.

Reply:
They're supported - you just need to indicate it with:

--filetypes:robot,resource,tsv,rst,rest 

(by default only robot and resource is parsed but you can configure whatever extension you want, even txt or png ;) )

Reply:
Writing robot tests in PNG would be something unusual 😄